lib/miio/Status.js

```diff
@@ -4,16 +4,17 @@
  */
 const Status = function() {
     /** List of allowed values as per Roborock.STATUS_MAP */
-    this.state = "UNKNOWN";
+    this.state = 0;
     this.error_code = null;
     this.battery = null;
     this.clean_time = null;   // Seconds?
     this.clean_area = null;   // mm²
     this.error_code = null;   // TODO
     this.map_present = false; // TODO
-    this.in_cleaning = false; // TODO
+    this.in_cleaning = 0; // TODO
     this.fan_power = null;
     this.dnd_enabled = false;
+    this.lab_status = 0;
```

Comment:
If adding a new field here, you should precisely document what it means.

Reply:
That was also a fix due to type checking.
There is Code which uses it as number, but I don't know what each number means.

Reply:
Looking at [client/settings-persistent-data.html#L102-L167](https://github.com/Hypfer/Valetudo/blob/master/client/settings-persistent-data.html), it seems like `1` means `persistent data enabled` and every other value means `persistent data disabled`.